blog/content/posts/2025-09-05-php-sdk.md

```diff
@@ -0,0 +1,29 @@
++++
+date = '2025-09-05T00:00:00Z'
+title = 'Announcing the Official PHP SDK for MCP'
+author = 'David Soria Parra (Lead Maintainer), Christopher Hertel (Symfony), Roman Pronskiy (PHP Foundation)'
+tags = ['announcement', 'community']
++++
+
+The official [PHP SDK](https://github.com/modelcontextprotocol/php-sdk) for the Model Context Protocol is now generally available. 
+
+Built in collaboration with the [PHP Foundation](https://thephp.foundation/) and [Symfony](https://symfony.com/), the PHP SDK handles protocol details, so developers don’t have to worry about low-level mechanics and can focus on building their applications. 
+
+The initial release enables PHP developers to build MCP [servers](https://modelcontextprotocol.io/docs/learn/server-concepts), exposing [tools](https://modelcontextprotocol.io/docs/learn/server-concepts#tools-ai-actions), [prompts](https://modelcontextprotocol.io/docs/learn/server-concepts#prompts-interaction-templates), and [resources](https://modelcontextprotocol.io/docs/learn/server-concepts#resources-context-data) to AI applications. Support for PHP applications to act as MCP [clients](https://modelcontextprotocol.io/docs/learn/client-concepts) will follow. 
```

Comment:
Would be worth including a section right below this that gives a very quick demo on how to build a MCP server with the PHP SDK. Doesn't need to be super-complex, just showcase the developer experience.